src/components/dashboard/sidebar/SubscriptionSummaryCard.jsx

```diff
@@ -30,46 +23,66 @@ const SubscriptionSummaryCard = ({
     subscriptionExpiringWarningModalOpen,
     onSubscriptionExpiringWarningModalClose,
   ] = useToggle(false);
-
+  const intl = useIntl();
   const badgeVariantAndLabel = useMemo(() => {
     if (subscriptionPlan) {
       if (subscriptionPlan.daysUntilExpiration <= SUBSCRIPTION_EXPIRED) {
         // Subscription has expired
         return ({
           variant: SUBSCRIPTION_EXPIRED_BADGE_VARIANT,
-          label: SUBSCRIPTION_EXPIRED_BADGE_LABEL,
+          label: intl.formatMessage({
+            id: 'enterprise.dashboard.sidebar.subscription.expired-badge.label',
+            defaultMessage: 'Expired',
+            description: 'Subscription expired badge label on the enterprise dashboard sidebar.',
+          }),
         });
       }
       if (programProgressPage && subscriptionPlan.daysUntilExpiration <= SUBSCRIPTION_DAYS_REMAINING_SEVERE) {
         // Expiration is approaching
         return ({
           variant: SUBSCRIPTION_WARNING_BADGE_VARIANT,
-          label: SUBSCRIPTION_EXPIRING_SOON_BADGE_LABEL,
+          label: intl.formatMessage({
+            id: 'enterprise.dashboard.sidebar.subscription.expiring-soon-badge-label',
+            defaultMessage: 'Expiring Soon',
+            description: 'Subscription expiring soon badge label on the enterprise dashboard sidebar.',
+          }),
         });
       }
       if (!programProgressPage && subscriptionPlan.daysUntilExpiration <= SUBSCRIPTION_DAYS_REMAINING_SEVERE) {
         // Expiration is approaching
         return ({
           variant: SUBSCRIPTION_WARNING_BADGE_VARIANT,
-          label: SUBSCRIPTION_WARNING_BADGE_LABEL,
+          label: intl.formatMessage({
+            id: 'enterprise.dashboard.sidebar.subscription.warning-badge.label',
+            defaultMessage: 'Expiring',
+            description: 'Subscription warning badge label on the enterprise dashboard sidebar.',
```

Comment:
Could we provide further details in the description to match the `defaultMessage`, thereby providing translator with additional context?
Something like: "Subscription warning badge label indicating that the subscription is expiring, on the enterprise dashboard sidebar"